src/pyhf/infer/test_statistics.py

```diff
@@ -31,7 +73,7 @@ def qmu(mu, data, pdf, init_pars, par_bounds):
         >>> init_pars = model.config.suggested_init()
         >>> par_bounds = model.config.suggested_bounds()
         >>> pyhf.infer.test_statistics.qmu(test_mu, data, model, init_pars, par_bounds)
-        3.938244920380498
+        array(3.93824492)
```

Comment:
This current example will raise the `qmu test statistic used for fit configuration with POI bounded at zero. Use qmutilde` warning, so we should change the example for one suited to `qmu`.

Reply:
done